libs/ui/src/components/NeoModal/NeoModal.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <o-modal
+    class="neo-modal"
+    :active.sync="isImageModalActive"
+    @close="updateClose">
+    <slot />
+  </o-modal>
+</template>
+
+<script lang="ts" setup>
+import { OModal } from '@oruga-ui/oruga'
+
+const props = defineProps<{
+  value: boolean
```

Comment:
Is it better to use name `visible/active`? wdyt

Reply:
The way that @prachi00 used that could be implemented with v-model

Reply:
Yes, I mean we could rename `value` to `active` for improved clarity.